xil_carma_basic_lanefollow_scenario/cdasim_config/.env

```diff
@@ -0,0 +1,7 @@
+START_DELAY_IN_SECONDS_1=15
+SPAWN_POINT_1="151.2,-305.9,0,0,0,0'"
+ROUTE_1="Release_test_case_1"
+
+START_DELAY_IN_SECONDS_2=15
```

Comment:
nit: For basic lane following scenario, it seems slightly overkill to have two vehicles configured by default. I think we can keep the second vehicle setup commented out, but will leave this up to you to decide.

Reply:
Okay I will comment out the 2nd vehicle